Client/mods/deathmatch/logic/CClientPlayer.cpp

```diff
@@ -88,6 +92,16 @@ CClientPlayer::CClientPlayer ( CClientManager* pManager, ElementID ID, bool bIsL
     // Add us to the player list
     m_pManager->GetPlayerManager ()->AddToList ( this );
 
+    // Check DisableCollisions //
+    if ( !m_DisabledCollisions.empty () )
+    {
+        std::list < CClientEntity * > ::iterator iter = m_DisabledCollisions.begin ();
```

Comment:
You added `using std::list;` on top of this translation unit, but kept the **std::** namespace here